deploy/docker/startup.sh

```diff
@@ -26,6 +29,18 @@ fi
 rmdir /srv/backup
 ln -s ${MNT_DIR}/backup /srv/backup
 
+# Replace the /var/log/apache2 in the image with a remotely-mounted one
+# Add a per-container piece because logs may be written during deployments
+APACHE_LOG_DIR="${MNT_DIR}/apache_logs/${HOSTNAME_LOCALIP_PART}"
+if [ ! -e "${APACHE_LOG_DIR}" ]; then
+  mkdir -p ${APACHE_LOG_DIR}
+  chown root:adm ${APACHE_LOG_DIR}
+  chmod 750 ${APACHE_LOG_DIR}
+fi
+mv /var/log/apache2/* ${APACHE_LOG_DIR}/
```

Comment:
Oh, huh, this is a ticking time bomb for when a private IP gets reused, which won't happen on dev sites, but will eventually happen on prod and staging.  Lemme make this `mv -f`.  (That's not very harmful to do because we only care about recent logs, and private IPs won't get reused very quickly; they just eventually will, and a deployment will fail mysteriously and we'll be sad.)